### ISSUE TITLE
2.1.4.1- Untranslated string

### ISSUE THREAD

**Issue by TheDutchJewel** [huhwt/huhwt-cce#3]
In version 2.1.4, an untranslated string `Webtrees Standard action` was added in `Execute an action on records > Download records ...`: 
![afbeelding](https://user-images.githubusercontent.com/46184729/171565387-8ecdb7fc-b743-40e5-9442-75e75d4e1b93.png)

**Comment by huhwt** [comment]
That's right, there's an adjustment that's slipped in that hasn't been finalized yet. Please do not add a new translation yet, the module is currently being revised/added/extended.

**Comment by TheDutchJewel** [comment]
By the way, also the `convert from UTF-8 to ISO-8859-1` is now untranslated, while is was translated in previous versions.
